frontend/dist/clouwaypush.js

```diff
@@ -1,355 +0,0 @@
-/**
- * clouwaypush - 2015-12-26
- *
- * Copyright (c) 2015 clouWay ltd
- */
-(function ( window, angular, undefined ) {
```

Comment:
I'll, but this need to be automated with a proper release cycle, cause everybody could forget that.